app/assets/javascripts/invite_users.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
The reason the Tealeaf courses recommend not to use generators is because of all these files that get left behind.